tests/geopage/models.py

```diff
@@ -161,70 +157,145 @@ class GeoPageWithLeaflet(Page):
 
 
 class GeoStreamPage(Page):
-    body = StreamField(
-        [
-            ("map", GoogleMapsBlock()),
-            ("map_with_leaflet", LeafletBlock()),
-            (
-                "map_struct",
-                blocks.StructBlock(
-                    [
-                        ("address", GeoAddressBlock(required=True)),
-                        ("map", GoogleMapsBlock(address_field="address")),
-                    ],
-                    icon="user",
+    if WAGTAIL_VERSION >= (3, 0):
```

Comment:
Hi @nickmoreton ,
This should be fixed by this commit: https://github.com/Frojd/wagtail-geo-widget/pull/111/commits/fc815b7633a5c100d777a4335ae9bb58f522a4fa

I have added the attribute inside the page or migration. This can also be outside the page so it can be shared by streamfields of other models, but there's only one instance per model file.

Reply:
Looks great 👍